evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -255,9 +255,9 @@ def round_dict_keys(dictionary, places=6):
     num_cat_features = len(set(X["currency"]))
     assert num_cat_features == 164
 
-    part_dep_ans = {0.1432616813857269: 154, 0.1502346349971562: 1, 0.14487916687594762: 1,
-                    0.1573183451314127: 1, 0.11695462432136654: 1, 0.07950579532536253: 1, 0.006794444792966759: 1,
-                    0.17745270478939879: 1, 0.1666874487986626: 1, 0.13357573073236878: 1, 0.06778096366056789: 1}
+    part_dep_ans = {0.19149451286750388: 154, 0.18249819137798806: 1, 0.24969722735684582: 1, 0.18542254073490247: 1,
```

Comment:
What happened here is that since we updated the `ComponentGraph` to preserve the woodwork types throughout, card_id, store_id, customer_present get preserved as ints/bool so the predictions are different which makes the partial dependence results different.

This is a snapshot from the debugger of the ww types as soon as they make it to estimator.fit. Note that the ohe-columns are double instead of bool because of the standard scaler. 

### Main
![woodwork_types_estimator_main](https://user-images.githubusercontent.com/41651716/120704076-20cd8880-c484-11eb-9a46-6e4e829b565c.png)


### This branch
![image](https://user-images.githubusercontent.com/41651716/120705134-70f91a80-c485-11eb-9d53-1c7dbdd12e3c.png)